lib/tasks/foreman_plugin_template_tasks.rake

```diff
@@ -24,15 +24,15 @@ namespace :test do
 end
 
 namespace :foreman_plugin_template do
-  task :rubocop do
+  task rubocop: :environment do
```

Comment:
This whole task is no longer needed since we load RuboCop via the `Rakefile` instead.